packages/Php/src/Rector/BinaryOp/IsCountableRector.php

```diff
@@ -52,10 +52,19 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
-        if (! $this->isAtLeastPhpVersion('7.3')) {
+        if ($this->shouldSkip()) {
             return null;
         }
 
-        return $this->dualCheckToAble->processBooleanOr($node, 'Countable', 'is_countable') ?: $node;
+        return $this->isArrayAndDualCheckToAble->processBooleanOr($node, 'Countable', 'is_countable') ?: $node;
+    }
+
+    private function shouldSkip(): bool
+    {
+        if ($this->isAtLeastPhpVersion('7.3')) {
```

Comment:
Shouldn't those check be opposite?

You can have polyfill with that method on even PHP 7.0, so first it should check for polyfill/method availability, then check PHP version, no?

Reply:
You're right of course!

Care for you PR?

Reply:
https://github.com/rectorphp/rector/pull/1677